{{cookiecutter.plugin_name}}/src/{{cookiecutter.module_name}}/_writer.py

```diff
@@ -25,7 +35,21 @@ def write_single_image(path: str, data: Any, meta: dict) -> List[str]:
 
 
 def write_multiple(path: str, data: List[FullLayerData]) -> List[str]:
-    """Writes multiple layers of different types."""
+    """Writes multiple layers of different types.
+    
+    Parameters
+    ----------
+    path : str
+    data : A layer tuple.
+           Tuple contains three elements: (data, kwargs, layer_type)
+           `data` is the layer data
+           `kwargs` is a dictionary of keyword arguments
```

Comment:
Should be updated to meta to have the same names, as is same as meta in previous function.